apps/api/src/app/bridge/usecases/store-control-variables/store-control-variables.usecase.ts

```diff
@@ -21,28 +18,22 @@ export class StoreControlVariablesUseCase {
     );
 
     if (!workflowExist) {
-      throw new ApiException('Workflow not found');
+      throw new NotFoundException('Workflow not found');
     }
 
     const step = workflowExist?.steps.find((item) => item.stepId === command.stepId);
 
     if (!step || !step._id) {
-      throw new ApiException('Step not found');
+      throw new NotFoundException('Step not found');
     }
 
-    const stepDefaultControls = defaults(
-      (step.template as any)?.controls?.schema || (step.template as any)?.inputs?.schema,
-      {}
-    ) as JsonSchema;
-
```

Comment:
Defaults are generated on `@novu/framework`, there is no need to generate defaults on the API.

Reply:
The JSON schema from `@novu/framework` is passed to the frontend to render the default values, so we don't need to create defaults on the API.